addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/handler/RFXComBridgeHandler.java

```diff
@@ -201,15 +201,6 @@ private synchronized void connect() {
             }
         } catch (IOException e) {
             logger.error("Connection to RFXCOM transceiver failed", e);
-            if ("device not opened (3)".equalsIgnoreCase(e.getMessage())) {
-                if (connector instanceof RFXComJD2XXConnector) {
-                    logger.info("Automatically Discovered RFXCOM bridges use FTDI chip driver (D2XX)."
-                            + " Reason for this error normally is related to operating system native FTDI drivers,"
-                            + " which prevent D2XX driver to open device."
-                            + " To solve this problem, uninstall OS FTDI native drivers or add manually universal bridge 'RFXCOM USB Transceiver',"
-                            + " which use normal serial port driver rather than D2XX.");
-                }
-            }
         } catch (Exception e) {
             logger.error("Connection to RFXCOM transceiver failed", e);
         } catch (UnsatisfiedLinkError e) {
```

Comment:
Can this exception still happen?

Reply:
You're right, I think it can no more happen, it was probably linked to JD2xx. I suppressed it.